backend/pkg/api/connect/interceptor/endpoint_check.go

```diff
@@ -47,7 +47,7 @@ func NewEndpointCheckInterceptor(cfg *config.ConsoleAPI, logger *zap.Logger) *En
 	}
 }
 
-// WrapUnary creates an interceptor to validate Connect requests.
+// WrapUnary creates an interceptor to validate KafkaConnect requests.
```

Comment:
I think this is not intentional ... "Connect" here refers to ConnectRPC

Reply:
Too much of connect in my head :D